src/routes/index/_about.svelte

```diff
@@ -88,7 +88,7 @@
 							contributions qualify for Appwrite Swag!
```

Comment:
can we maybe say,  "....contributions quality for digital Appwrite Swag! ....

Reply:
How is it now?